web/source/dom/domEventHandlers.ts

```diff
@@ -563,8 +566,9 @@ namespace com.keyman.dom {
 
       // Determines the actual TouchAliasElement - the part tied to an OutputTarget.
       // Ideally, we shouldn't need the second part as a fallback; it's there to preserve existing
-      // behavior from 13.0, as this was refactored made LATE in the 14.0 beta process.
-      let target = findTouchAliasTarget(tTarg) || (tTarg as TouchAliasElement);
+      // behavior from 13.0, as this was refactored LATE in the 14.0 beta process.
+      let target = findTouchAliasTarget(tTarg);
```

Comment:
The current logs we see for #4797 were likely from here; failure to find a `TouchAliasElement` from `tTarg` would generally lead to `setTextCaret` being `undefined` as seen in the logs.

So, this _removes_ that fallback.

Reply:
Doesn't this change make the comment above incorrect?

Reply:
Right, I should remove the

> Ideally, ...

part when this goes through.